README.md

```diff
@@ -1,61 +1,62 @@
 [![Build Status](https://dev.azure.com/knotx/Knotx/_apis/build/status/Knotx.knotx-starter-kit?branchName=master)](https://dev.azure.com/knotx/Knotx/_build/latest?definitionId=3&branchName=master)
 
 # Knot.x Starter Kit
-Knot.x Starter Kit is a template project that simplifies Knot.x project setup. It allows you to 
+Starter Kit is a template project that simplifies Knot.x project setup. It allows you to 
 customize the [Knot.x distribution](https://github.com/Knotx/knotx-stack) with your own modules and
-configuration entries. Then it builds and validates your custom Docker image. 
+configuration entries. 
 
+Starter Kit artefacts (see the build/distributions folder) are: 
+- a ZIP file when `./gradlew build-stack` or `./gradlew build`
+- a custom Docker image when `./gradlew build-docker`
 
-## How to build
+### Prerequisites
+Follow [Development process](https://github.com/Knotx/knotx-aggregator) instructions to build all 
+SNAPSHOT artefacts.
+
+## How to build & run
 To start a new Knot.x project simply download the chosen ZIP version from https://github.com/Knotx/knotx-starter-kit/tags.
-To use the latest DEV version download this [ZIP](https://github.com/Knotx/knotx-starter-kit/archive/master.zip) and [switch to SNAPSHOT version](https://github.com/Knotx/knotx-starter-kit#how-to-work-with-snapshot).
 
+### ZIP file
 In the unziped folder run
 ```
-$> ./gradlew build
+$> ./gradlew build-stack
 ```
-
 to:
 - build all your custom Knot.x modules
-- prepare your custom Docker image with all required dependencies (including your custom modules and 
-its transitive dependencies)
-- validate your Docker image with system tests
+- download Knot.x Stack artefact, add all required dependencies (including your custom modules and 
+its transitive dependencies), override configs with `/knotx/config`
+- build the custom ZIP artefact
 
-## How to run
-Start Docker container
+Then, go to the `build/distributions` directory, unzip the ZIP artefact, start Knot.x:
 ```
-docker run -p8092:8092 knotx/knotx-starter-kit
+chmod +x bin/knotx
+bin/knotx run-knotx
 ```
 
 and validate urls:
 - [localhost:8092/api/v1/example](http://localhost:8092/api/v1/example)
 - [localhost:8092/api/v2/example](http://localhost:8092/api/v2/example)
 
-## How to debug
-
-#### Normal debugging
-Edit `Dockerfile` in `docker` folder by adding
-```dockerfile
-RUN sed -i 's/# JVM_DEBUG=/JVM_DEBUG=/g' /usr/local/knotx/bin/knotx
+### Docker
+In the unziped folder run
```

Comment:
`unzipped`

Reply:
what unzipped folder are we talking about here? the `zip` that was generated during `build-stack` ? if so, shouldn't we make it independent of building the stack itself? so that `build-docker` would be enough?

Reply:
Fixed documentation, I mean the Starter Kit zip file.